timetracking/nginx.conf

```diff
@@ -0,0 +1,15 @@
+server {
+    listen 80;
+    server_name localhost;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri /index.html;
+    }
+
+    error_page   500 502 503 504  /50x.html;
```

Comment:
do we have files in the built bundle that matches the pattern /50x.html ?

Reply:
removed the file